.github/workflows/main.yml

```diff
@@ -0,0 +1,202 @@
+name: Litmus-CI
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  tests:
+    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/run-e2e')
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Notification for e2e Start
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: "${{ github.event.comment.id }}"
+          body: |
+            ****
+            **Test Status:** The e2e test has been started please wait for the test result ...      
+
+      #Using the last commit id of pull request
+      - uses: octokit/request-action@v2.x
+        id: get_PR_commits
+        with:
+          route: GET /repos/:repo/pulls/:pull_number/commits
+          repo: ${{ github.repository }}
+          pull_number: ${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: set commit to output
+        id: getcommit
+        run: | 
+           prsha=$(echo $response | jq '.[-1].sha'  | tr -d '"')
+           echo "::set-output name=sha::$prsha" 
+        env: 
+          response:  ${{ steps.get_PR_commits.outputs.data }}
+          
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{steps.getcommit.outputs.sha}}      
+
+      - name: Publish to Docker Repository
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: litmuschaos/ansible-runner
+          username: ${{ secrets.DUSER }}
+          password: ${{ secrets.DPASS }}
+          dockerfile: build/ansible-runner/Dockerfile
+          tags: "ci"
```

Comment:
Is this needed here? i.e.., we build, but do not push here right?

Reply:
The tag should not be `ci` here.
@ksatchit  actually the job image pull policy is `Always` by default so I've to push the image.

https://github.com/litmuschaos/litmus/blob/234c5964b083cc2e2d21f671d4b0a0adcfd6a716/experiments/generic/container_kill/container_kill_k8s_job.yml#L17

Reply:
Do you mean the litmus job? If so, we can get this overridden.